<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyWindow</class>
 <widget class="QWidget" name="StudyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="wordLabel">
     <property name="text">
      <string>Word</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 20pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
    <item>
     <widget class="QComboBox" name="modeComboBox">
      <property name="editable">
        <bool>false</bool>
      </property>
      <item>
        <property name="text">
         <string>Flashcard</string>
        </property>
      </item>
      <item>
        <property name="text">
         <string>Multiple Choice</string>
        </property>
      </item>
     </widget>
    </item>
   <item>
    <widget class="QPushButton" name="revealButton">
     <property name="text">
      <string>Reveal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="definitionLabel">
     <property name="text">
      <string>Definition</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
    <item>
     <widget class="QWidget" name="choicesWidget">
      <layout class="QGridLayout" name="choicesGrid">
        <item row="0" column="0">
         <widget class="QPushButton" name="choiceButton1">
          <property name="text">
            <string>Choice 1</string>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="choiceButton2">
          <property name="text">
            <string>Choice 2</string>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="choiceButton3">
          <property name="text">
            <string>Choice 3</string>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="choiceButton4">
          <property name="text">
            <string>Choice 4</string>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
         </widget>
        </item>
      </layout>
     </widget>
    </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="againButton">
       <property name="text">
        <string>Again</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hardButton">
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goodButton">
       <property name="text">
        <string>Good</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="easyButton">
       <property name="text">
        <string>Easy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
